wordpress-mobile/WordPress-iOS
# Fix: Notice overlap with SiteIconActionSheet

Fixes https://github.com/wordpress-mobile/WordPress-iOS/issues/16465

To test:
1. On Update the site title. 
2. Add the site icon. 
Notice that the site title update notice doesn't over lap and interfere with the action that you wanted todo.
3. Do the same thing but now that you have the site icon. Notice that that the site title update notice doesn't interfere with the action that you wanted to do. 

Screenshots

https://user-images.githubusercontent.com/115071/119584709-00177b80-bd7e-11eb-9f97-4c3d61a246a4.mp4

IPad
<img width="1748" alt="Screen Shot 2021-05-25 at 5 23 39 PM" src="https://user-images.githubusercontent.com/115071/119584729-0a397a00-bd7e-11eb-95c3-529d730c1a59.png">


## Regression Notes
1. Potential unintended areas of impact
N/A

2. What I did to test those areas of impact (or what existing automated tests I relied on)


3. What automated tests I added (or what prevented me from doing so)
None

PR submission checklist:
